doc/faq.rst

```diff
@@ -33,29 +95,34 @@ Attachments are visible to users of non Autocrypt-compatible MUAs,
 while headers are not.  We don't want to present distracting or
 confusing material to those users.
 
+
+
 Why are you sending keys in all the mails and not just announcing capabilities?
 -------------------------------------------------------------------------------
 
-We did this in a previous version. We decided against it because it
-requires the MUA to keep the information who announced Autocrypt and
-who they requested keys from.
+We played with capabilities in a previous design. We decided against them because
+they require the MUA to keep the information who announced Autocrypt and who
+they requested keys from and that is complicated for multi-device settings, in particular.
 
 
-Why RSA3072 and 25519 only later?
----------------------------------
+Why also RSA3072 and not only Curve 25519 keys?
```

Comment:
Just for the record, this is also answered in a note in the spec itself: https://autocrypt.org/level1.html#openpgp-based-key-data